go_lib/deckhouse-config/status.go

```diff
@@ -58,11 +60,6 @@ func (s *StatusReporter) ForConfig(cfg *d8v1alpha1.ModuleConfig, bundleName stri
 		}
 	}
 
-	moduleType := "Embedded"
-	if isExternal {
-		moduleType = fmt.Sprintf("External: %s", repo)
-	}
```

Comment:
It will break [this part](https://github.com/deckhouse/deckhouse/blob/main/modules/002-deckhouse/hooks/pkg/apis/v1alpha1/module.go#L86-L91) of a Module generation.
As result, it will break [the cleanup](https://github.com/deckhouse/deckhouse/blob/main/modules/005-external-module-manager/hooks/cleanup_releases.go#L54)

Reply:
Or not...
Anyway, we have one more place with an `external type` labels

Reply:
Good catch! Fixed.